test/job_queue_test.exs

```diff
@@ -64,7 +64,7 @@ defmodule EctoJob.JobQueueTest do
   end
 
   describe "JoqQueue.activate_scheduled_jobs" do
-    test "Updates scheduled job to active" do
+    test "Updates scheduled job to AVAILABLE" do
```

Comment:
I found that when I was reading these tests to get acquainted with the codebase that the terminology was a touch confusing. I just edited for consistency.